### PR TITLE
Improve test coverage for VaultAdmin.sol

### LINKED ISSUE #716: Coverage Report - Test VaultAdmin.sol
Test the following functions with the described scenario

- [x] withValidPercentage, when percentage is greater than FixedPoint.ONE
- [x] success when calling pauseVault
- [x] success when calling unpauseVault
- [x] _setVaultPaused 
    - [x] when pausing is true and vault is already paused 
    - [x] when pausing is false and vault is already unpaused
    - [x] when pausing but vaultWindowPauseEndTime has expired
- [x] _setPoolPaused 
    - [x] when pausing is true and pool is already paused 
    - [x] when pausing is false and pool is already unpaused
- [ ] updateAggregateSwapFeePercentage (onlyProtocolFeeController)
- [ ] updateAggregateYieldFeePercentage (onlyProtocolFeeController)
- [ ] setProtocolFeeController
- [ ] disableRecoveryMode
- [ ] addLiquidityToBuffer (when underlying asset != _bufferAssets[IERC20(address(wrappedToken))])

### PR DESCRIPTION
**Pull request by joaobrunoah**
# Description

This PR improves the test coverage for VaultAdmin.sol. To identify what should be tested, I ran `yarn coverage` inside vault package.

Note: If hardhat and forge tests are merged, the branch coverage report breaks. So, I adapted `coverage.sh` to not consider hardhat input.

## Type of change

- [ ] Bug fix 
- [ ] New feature 
- [ ] Breaking change 
- [ ] Dependency changes
- [ ] Code refactor / cleanup
- [ ] Documentation or wording changes
- [x] Other

## Checklist:

- [x] The diff is legible and has no extraneous changes
- [x] Complex code has been commented, including external interfaces
- [x] Tests have 100% code coverage
- [x] The base branch is either `main`, or there's a description of how to merge

## Issue Resolution

Closes #716

### CODE REVIEW

File: pkg/vault/test/foundry/unit/VaultCommonModifiers.t.sol [hunk 1]
```diff
@@ -1,5 +1,9 @@
+// SPDX-License-Identifier: GPL-3.0-or-later
+
 pragma solidity ^0.8.24;
 
+import "forge-std/Test.sol";
```
Comment (elshan-eth) [inline pkg/vault/test/foundry/unit/VaultCommonModifiers.t.sol:5]: Seems like unnecessary import

File: pkg/vault/test/foundry/unit/VaultAdminUnit.t.sol [hunk 2]
```diff
@@ -0,0 +1,114 @@
...
+        vault.manualSetPoolConfig(pool, poolConfig);
+
+        // Pool is already paused and we're trying to pause again.
+        vm.expectRevert(abi.encodeWithSelector(IVaultErrors.PoolPaused.selector, pool));
+        vault.manualPausePool(pool);
+    }
+
+    function testUnpausePoolWhenPoolIsUnpaused() public {
+        // Only internal functions are used, so the pool does not need to be registered.
+        address pool = address(0x123);
```
Comment (elshan-eth) [inline pkg/vault/test/foundry/unit/VaultAdminUnit.t.sol:85]: Maybe it will be better if the pool var will be constant

**Review by elshan-eth — APPROVED** [review]
LGTM! Just few comments

File: pkg/vault/test/foundry/unit/VaultCommonModifiers.t.sol (hunk 1 above)
Comment (joaobrunoah) [reply pkg/vault/test/foundry/unit/VaultCommonModifiers.t.sol:5]: It is, but I understand it as a pattern in test files. Not sure if we should stick to this pattern or not, but it makes it easy to console.log stuff, for example, when we need to debug something.

File: pkg/vault/test/foundry/unit/VaultAdminUnit.t.sol [hunk 3]
```diff
@@ -0,0 +1,152 @@
...
+    // withValidPercentage
+    function testWithValidPercentage() public view {
+        // Any percentage below 100% is valid. This test expects not to be reverted.
+        vault.mockWithValidPercentage(0.5e18);
+    }
+
+    function testWithValidPercentageRevert() public {
+        // Any percentage above 100% is not valid and modifier should revert.
+        vm.expectRevert(IVaultErrors.ProtocolFeesExceedTotalCollected.selector);
+        vault.mockWithValidPercentage(1.5e18);
```
Comment (jubeira) [inline pkg/vault/test/foundry/unit/VaultAdminUnit.t.sol:46]:
```suggestion
        vault.mockWithValidPercentage(1e18 + 1);
```

**Review by jubeira — APPROVED** [review]
LGTM, great job! Thanks @joaobrunoah

**Pull request merged**